kie-server-parent/kie-server-tests/kie-server-integ-tests-controller/src/test/java/org/kie/server/integrationtests/controller/KieControllerManagementIntegrationTest.java

```diff
@@ -425,9 +425,12 @@ public void testStartAndStopContainer() throws Exception {
 
     @Test
     public void testStartNotExistingContainer() throws Exception {
+        ContainerSpec containerSpec = new ContainerSpec();
+        containerSpec.setId(CONTAINER_ID);
+        containerSpec.setServerTemplateKey(new ServerTemplateKey(kieServerInfo.getServerId(), null));
         // Try to start not existing container using kie controller without created kie server instance.
         try {
-            mgmtControllerClient.startContainer(kieServerInfo.getServerId(), CONTAINER_ID);
+            mgmtControllerClient.startContainer(containerSpec);
```

Comment:
is this really better api - to accept ContainerSpec instead of two strings? Just asking as to be able to start container you need these two strings anyway and then wrap them with two additional objects - ContainerSpec and ServerTemplateKey... feel free to ignore just thinking at loud :)